src/types/onyx/OriginalMessage.ts

```diff
@@ -144,20 +144,20 @@ type OriginalMessageReportPreview = {
     };
 };
 
+type CommonOriginalMessage = {
```

Comment:
"Common" between what? This context is lost right after this symbol is exported.

Maybe `ChangeLogOriginalMessage`? (as it's common between "policy _changelog_" and "room _changelog_" types)

I know that it's a bit confusing that `OriginalMessagePolicyChangeLog` itself contains a property `originalMessage`, but it's beyond the scope of this PR.